components/dashboard/src/admin/License.tsx

```diff
@@ -0,0 +1,215 @@
+                        <p className="dark:text-gray-500 pt-2 font-semibold">License Type</p>
+                        <h4 className="dark:text-gray-300 pt-1 text-lg">{capitalizeInitials(license?.type || "")}</h4>
+                        <div className="flex justify-end bottom-2 relative">
+                            <button
+                                type="button"
+                                onClick={(e) => {
+                                    e.preventDefault();
+                                    window.location.href = "https://www.gitpod.io/self-hosted";
+                                }}
+                            >
+                                <div className="font-semibold">Compare Plans</div>
+                            </button>
+                        </div>
+                    </SolidCard>
+                </div>
+            </PageWithSubMenu>
+        </div>
+    );
+}
+
+function capitalizeInitials(str: string): string {
+    return str
+        .split("-")
+        .map((item) => {
+            return item.charAt(0).toUpperCase() + item.slice(1);
+        })
+        .join(" ");
+}
+
+function getSubscriptionLevel(license: LicenseInfo): ReactElement[] {
+    switch (license.plan) {
+        case "prod":
+        case "trial":
+            return professionalPlan(license.userCount || 0, license.seats, license.plan == "trial", license.validUntil);
+        case "community":
+            return communityPlan(license.userCount || 0, license.seats, license.fallbackAllowed);
+        default: {
+            return defaultMessage();
+        }
+    }
+}
+
+function licenseLevel(level: string): ReactElement {
+    return <p className="text-white dark:text-black font-bold pt-4"> {level}</p>;
+}
+
+function additionalLicenseInfo(data: string): ReactElement {
+    return <p className="dark:text-gray-500">{data}</p>;
+}
+
+function defaultMessage(): ReactElement[] {
+    const alertMessage = () => {
+        return (
+            <>
+                <div>Inactive or unknown license</div>
+                <div className="flex justify-right my-4 mx-1">
+                    <Alert fill="grey" className="h-8 w-8" />
+                </div>
+            </>
+        );
+    };
+
+    return [licenseLevel("Inactive"), additionalLicenseInfo("Free"), alertMessage()];
+}
+
+function professionalPlan(userCount: number, seats: number, trial: boolean, validUntil: string): ReactElement[] {
+    const alertMessage = (aboveLimit: boolean) => {
+        return aboveLimit ? (
+            <>
+                <div className="text-red-600">You have exceeded the usage limit.</div>
+                <div className="flex justify-right my-4 mx-1">
+                    <Alert fill="red" className="h-6 w-6" />
+                </div>
+            </>
+        ) : (
+            <>
+                <div className="text-green-600">You have an active professional license.</div>
```

Comment:
**nitpick**: Adding below a different green color to use on the dark theme following the initial specs.

```suggestion
                <div className="text-green-600 dark:text-green-400">You have an active professional license.</div>
```